src/components/CityCards.js

```diff
@@ -0,0 +1,35 @@
+import londonImage from '../images/london.jpg';
+import glasgowImage from '../images/glasgow.jpg';
+import manchesterImage from '../images/manchester.jpg';
+
+const cities = [
```

Comment:
This is a nice clean way of separating the data, but the exercise asks you to pass this data as a prop to `TouristInfoCards`. Could you update this?

Reply:
I have updated it, thanks.